py/strato/common/log/log2text.py

```diff
@@ -219,10 +222,18 @@ def printLogs(logFiles, formatter):
         currentLines[nextStreamId] = _getNextParsableEntry(inputStream[0], inputStream[1], _getColorCode(nextStreamId),
                                                            formatter)
 
+def updateConfFile(field, value):
+    with open(LOG_CONFIG_FILE_PATH, 'r') as f:
+        conf = yaml.load(f.read())
+        conf[field] = value
+    with open('/tmp/stratolog-conf.tmp', 'w') as f:
+        f.write(yaml.dump(conf))
+    os.system('sudo mv /tmp/stratolog-conf.tmp %s' % LOG_CONFIG_FILE_PATH)
+
 if __name__ == "__main__":
     import argparse
     parser = argparse.ArgumentParser()
-    parser.add_argument("logFiles", metavar='logFile', nargs='+', help='logfiles to read or - for stdin')
+    parser.add_argument("logFiles", metavar='logFile', nargs='*', help='logfiles to read or - for stdin')
```

Comment:
if you want to just update the config file, you won't have to pass positional arg